### DISCUSSION TITLE
no more patch releases?

### DISCUSSION THREAD

**Discussion by gennff** [PostgREST/postgrest#2289] [Q&A] [answered]
hi there, are you planning to provide patch releases further on? 9.0.0 stable is of novenber 21, couple of bug fixes since then, all tagged pre release. there have been patches for v5, 6, and 7. shall one stick to 9.0.0 stable for production use? thanks, peter

**Comment by steve-chavez** [comment]
Hello,

> there have been patches for v5, 6, and 7. 

Right, the problem is that we have some [breaking changes](https://github.com/PostgREST/postgrest/blob/main/CHANGELOG.md#changed) already - so we cannot do a patch release.

We had a discussion about this on https://github.com/PostgREST/postgrest/pull/2052#issuecomment-1030690124, so we'll compromise to always release a patch(or minor) version starting from the next.

---

What's blocking a release right now is implemeting https://github.com/PostgREST/postgrest/issues/2277#issuecomment-1127914913, which we need for offering a migration path to the new version(which will be v10).

**Comment by steve-chavez** [comment, marked as answer]
We made a new patch release: https://github.com/PostgREST/postgrest/releases/tag/v9.0.1
